components/Post/Post.js

```diff
@@ -64,7 +63,7 @@ function PostActions({ postId, is_post_liked, isExpanded }) {
       <div className={styles.likeShareTelegramIcons}>
         <LikeButton
           is_post_liked={is_post_liked}
-          onClick={() => togglePostLike(postId)}
+          onClick={() => postLike(postId, !is_post_liked)}
```

Comment:
This is out of scope. Let's leave it this time, but next time please create another tiny PR for such things.

Reply:
I think this is from the rebase.